ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/websocket/WebSocketService.java

```diff
@@ -197,4 +205,8 @@ public InetSocketAddress socketAddress() {
   private String socketAddressAsString(final SocketAddress socketAddress) {
     return String.format("host=%s, port=%d", socketAddress.host(), socketAddress.port());
   }
+
+  private String getToken(final ServerWebSocket websocket) {
```

Comment:
Maybe rename this to getAuthToken() or something similar?

Reply:
👍